### PR TITLE
logname: add page

### PR DESCRIPTION
**Pull request by ckauf**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by ckauf** [comment]
This is for issue #2213

File: pages/linux/logname.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# logname 
+
+> Shows the users login name.
```
Comment (sbrl) [inline pages/linux/logname.md]: Perhaps `user's` instead of `users` would read better?

File: pages/linux/logname.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- Display login user name:
```
Comment (sbrl) [inline pages/linux/logname.md]: Perhaps `Display the currently logged in user's name` would read better?

File: pages/linux/logname.md (hunk 1 above)
Comment (ckauf) [reply pages/linux/logname.md]: Yup. Done that.

File: pages/linux/logname.md (hunk 2 above)
Comment (ckauf) [reply pages/linux/logname.md]: Yup. Done that.

**Pull request merged**